x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinator.java

```diff
@@ -186,6 +187,7 @@ synchronized void updateStats(List<AutoFollowResult> results) {
                             "pattern [{}]", entry.getKey(), result.autoFollowPatternName), entry.getValue());
                     } else {
                         numberOfSuccessfulIndicesAutoFollowed++;
+                        recentAutoFollowErrors.remove(result.autoFollowPatternName + ":" + entry.getKey().getName());
```

Comment:
nit: can we compute the `key` once and use it to `put` or `remove` the recent auto follow error?